src/components/Content/Card.js

```diff
@@ -20,7 +20,7 @@ import { Image } from '../Image/Image';
 import { fontSizes, fontWeight } from '../Fonts/Fonts';
 //create constant for resued sizes
 
-export const Card = () => {
+export const Card = ({ data }) => {
```

Comment:
optional: you can consider destructuring the input
```
export const Card = ({ title, date, time }) => {
```

Reply:
this line is one of reasons why typescript will work better.
if you are using ts instead js
```
 export const Card = ({ title: string, date: Date, time: Date }) => {
```
with it, you have up to date document about function inputs

Reply:
Successfully desturctured